modules/doc/content/newsletter/2021_11.md

```diff
@@ -43,4 +43,17 @@ If any issues with the new packages are experienced, please open a new
 
 ## libMesh-level Changes
 
+- Exodus IGA fix for multiblock files
```

Comment:
We really don't want this to be a boring changelog...

Reply:
Version 2 is pushed.  It's at least less terse and less mechanical sounding; if it's not sufficiently less boring then in Version 3 I can put more dumb jokes into the commit rather than just the commit message.